src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/AbaqusHexahedronWriter.cpp

```diff
@@ -199,14 +199,7 @@ int32 writeElset(AbaqusHexahedronWriter* filter, const std::string& fileName, si
   fprintf(f, "**\n** Each Grain is made up of multiple elements\n**");
 
   // find total number of Grain Ids
-  int32 maxGrainId = 0;
-  for(usize i = 0; i < totalPoints; i++) // find number of grainIds
-  {
-    if(featureIds[i] > maxGrainId)
-    {
-      maxGrainId = featureIds[i];
-    }
-  }
+  int32 maxGrainId = *std::max_element(std::begin(featureIds), std::end(featureIds));
```

Comment:
```suggestion
  int32 maxGrainId = *std::max_element(featureIds.cbegin(), featureIds.cend());
```
Generally I'd say to use the member `begin` and `end` when we know the exact type.